AutoTrader/src/AutoTrader.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<Version>0.0.1</Version>
+		<TargetFrameworks>net472</TargetFrameworks>
+		<Platforms>x64</Platforms>
+		<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+		<LangVersion>10.0</LangVersion>
+		<Nullable>enable</Nullable>
+		<BuildForWindows>false</BuildForWindows>
+		<BuildForWindowsStore>false</BuildForWindowsStore>
+		<ModuleId>$(MSBuildProjectName)</ModuleId>
+		<ModuleName>$(MSBuildProjectName)</ModuleName>
+		<ModuleUrl></ModuleUrl>
+		<GameFolder>D:\SteamLibrary\steamapps\common\Mount &amp; Blade II Bannerlord</GameFolder>
```

Comment:
Also hardcoded install path here. I mean it would be reasonable to require a contributor to adjust that but then an addition to the README would be nice